dipy/data/fetcher.py

```diff
@@ -592,16 +595,19 @@ def read_syn_data():
 """
 
 
-def read_mni_template(contrast="T2"):
+def read_mni_template(version="a", contrast="T1"):
```

Comment:
A very valid point, the default value was changed because the version c
does not have a T2 image, so to rectify this we can either rechange the
default to T2 and it will throw a ValueError when version=c.
Or we can also include T2 for version c in the fetcher that way the default
can remain same.

On 10-Aug-2016 7:44 pm, "Jean-Christophe Houde" <notifications@github.com>
wrote:

In dipy/data/fetcher.py
<https://github.com/nipy/dipy/pull/1106#discussion_r74251872>:

> @@ -592,16 +595,19 @@ def read_syn_data():
>  """
>
>
> -def read_mni_template(contrast="
T2"):
> +def read_mni_template(version="a", contrast="T1"):

I know this PR is already merged, but isn't it a bad idea to silently
change the default value of some parameter like this? Someone calling this
from a script without any parameter might suddently encounter bizarre
results...

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/nipy/dipy/pull/1106/files/8499605f20b135f1ae80cf1020bcd75e55eafbd5#r74251872>,
or mute the thread
<https://github.com/notifications/unsubscribe-auth/AHkvV7xWulDpA1_vJ6ViATXpUBm3-Q23ks5qedyqgaJpZM4JahiD>
.